viostor/viostor.inx

```diff
@@ -97,7 +97,7 @@ HKR, "Parameters", "BusType", %REG_DWORD%, 0x00000001
 HKR, "Interrupt Management",, 0x00000010
 HKR, "Interrupt Management\MessageSignaledInterruptProperties",, 0x00000010
 HKR, "Interrupt Management\MessageSignaledInterruptProperties", MSISupported, 0x00010001, 1
-HKR, "Interrupt Management\MessageSignaledInterruptProperties", MessageNumberLimit, 0x00010001, 256
+HKR, "Interrupt Management\MessageSignaledInterruptProperties", MessageNumberLimit, 0x00010001, 257
```

Comment:
Why not increase it to the maximum?
https://github.com/virtio-win/kvm-guest-drivers-windows/blob/5c54cacb2ed054f9737fdadb84b3e4cd235a9928/NetKVM/netkvm-base.txt#L37